README.md

```diff
@@ -102,15 +102,16 @@ colorMode: {
   cookie: {
     key: 'nuxt-color-mode',
     options: {
-      path: nuxt.options.router.base // https://nuxtjs.org/api/configuration-router#base
+      path: nuxt.options.router.base, // https://nuxtjs.org/api/configuration-router#base
+      sameSite: 'Lax' // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
```

Comment:
Even though the cookie package will handle it fine, the value should be lower-case to match the types (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cookie/index.d.ts#L87)

(same in readme.md)